### PR TITLE
[tests] Reduce ionic fixture sizes to improve test timing

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
This PR regenerates the ionic-angular and ionic-react fixtures to be significantly smaller and simpler. 

I don't know why the lock files are so massive. If anyone has ideas to reduce the size of those (I've tried ripping out a bunch of dependencies).

### CODE REVIEW

**Review by styfle — DISMISSED** [review]

**Review by styfle — DISMISSED** [review]

File: packages/static-build/test/fixtures/50-ionic-react/yarn.lock
Comment (styfle) [inline packages/static-build/test/fixtures/50-ionic-react/yarn.lock:1]: Please keep the lock file because that will speed up deployments significantly

Comment (styfle) [reply packages/static-build/test/fixtures/50-ionic-react/yarn.lock:1]: Oh I see its `package-lock.json` now 👍

**Review by styfle — DISMISSED** [review]

**Review by trek — DISMISSED** [review]

**Comment by Ethan-Arrowood** [comment]
New test run timings:

- 50-ionic-react: 58685ms (34.0529% decrease)
- 52-ionic-angular: 69697ms (56.2213% decrease)

File: packages/static-build/test/fixtures/52-ionic-angular/vercel.json [hunk 1]
```diff
@@ -8,5 +8,5 @@
       }
     }
   ],
-  "probes": [{ "path": "/", "mustContain": "Ionic Conference App" }]
+  "probes": [{ "path": "/", "mustContain": "Ionic Angular on Vercel" }]
```
Comment (styfle) [inline packages/static-build/test/fixtures/52-ionic-angular/vercel.json:11]: In the future, we should remove `vercel.json` in favor of `probes.json`

**Review by styfle — APPROVED** [review]

**Review by cb1kenobi — APPROVED** [review]
Looks good as long as the tests pass. Nice work!

**Comment by styfle** [comment]
Looks like github actions are down. Might need to push an empty commit

**Pull request merged**